docs/adr/lts-strategy.md

```diff
@@ -0,0 +1,176 @@
+dependency and middleware packages, we must also address those packages as well. 
+
+### ADR Goals:
+
+- Improve documentation on our support goals
+- Document criteria and reasoning for when to release new majors
+- Define life cycle for releases, terminology we will use, 
+- Improve documentation on how our LTS relates to Node.js version support
+- Documentation on how our LTS relates to our direct dependencies and other Express project repos/packages
+- Set clear next steps for documenting and publicising the LTS strategy 
+- Select dates for known release line support and EOL
+- Create starter language and graphics for our future blog posts, website additions, and other necessary docs 
+
+### ADR Non-Goals:
+
+- Create an end-user documentation resource (either for immediate use or future use)
+- Commit to future release dates
+
+**Existing issues/discussions/pull requests related to this:**
+
+- Initial LTS strategy doc: https://github.com/expressjs/discussions/blob/master/docs/LTS-strategy.md
+- Additional LTS doc proposal: https://github.com/expressjs/admin/pull/3
+- https://github.com/ctcpip/express-release
+- Some other related discussions:
+  - https://github.com/expressjs/discussions/issues/196
+  - https://github.com/expressjs/discussions/issues/172
+  - https://github.com/expressjs/discussions/issues/67
+- TC Slack Discussion: https://openjs-foundation.slack.com/archives/C06KMCETHG9/p1724863852681889
+
+### Package/Repository Types
+
+We maintain four primary types of packages/repos:
+
+1. `express`: The minimal http framework
+2. `express` dependencies: Libraries which are used directly or transitively within `express` and also by others outside
+   of the project
+3. Middleware: Libraries for use *with* `express` but not installed with it directly
+4. Others: Tools, documentation, websites, resources, etc.
+
+While much of this proposal is targeted specifically at `express`, this terminology will be used to differentiate where
+we are not only talking about `express`.
+
+## Decision
+
+The Express project will have a well defined Long Term Support (LTS) strategy and schedule based on the following
+requirements:
+
+- `express` major versions will go through three supported phases: Current, Active, & Maintenance
+  - `CURRENT`: A new major version is designated as `CURRENT` upon release. It is available but not the `latest` version
+    on npm for a minimum of 3 months.
+  - `ACTIVE`: After the minimum 3 month period and the TC agrees it is stable and secure, the `ACTIVE` version is
+    tagged `latest` on npm for a minimum of 12 months.
+  - `MAINTENANCE`: When a new major version becomes `ACTIVE`, the previous major version enters `MAINTENANCE` for 12 months.
+- After the `MAINTENANCE` time has ended the major version is considered `EOL` and is unsupported.
+- During the 12 month `ACTIVE` period, a new major version may be released but *not* until at least 6 months have passed
```

Comment:
Earlier the period is defined as "at least 12". Not sure if the intention here is to mirror it or say exact 12

Reply:
Most of this was copied from conversations we had, so it was not all written at once and I probably should have re-read some after it was together. For this one, I actually think maybe we should just drop the months here so we dont have inconsistent language but also make clear it can happen during that period:

```suggestion
- During the `ACTIVE` period, a new major version may be released but *not* until at least 6 months have passed
```